### DISCUSSION TITLE
Question and answer to get Galaxy Brain batch

### DISCUSSION THREAD

**Discussion by dunkelhai** [community/community#33446] [Other Feature Feedback, Questions, & Ideas] [answered]
Question and answer to get Galaxy Brain batch

Can someone help me get please?

**Comment by samuel-ma** [comment, marked as answer]
@dunkelhai 
I'll gladly help out
Inbox...

**Reply by dunkelhai** [reply]
Thank you so much samuel :D 

Sorry but where's the function to chat here on GitHub?

**Comment by github-staff** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
